scripts/check_e2e_tests_are_captured_in_ci.py

```diff
@@ -25,10 +25,10 @@
 import python_utils
 import utils
 
-# These test suites are not present in travis CI. One is extra
+# These test suites are not present in Action. One is extra
 # (ie. (full: [*.js])) and other test suites are being run by CircleCI.
-TEST_SUITES_NOT_RUN_ON_TRAVIS = [
-    'full', 'accessibility', 'adminPage', 'classroomPage',
+TEST_SUITES_NOT_RUN_ON_ACTIONS = [
```

Comment:
Could we maybe say 'NOT_RUN_IN_CI'? Here and elsewhere, it may be advantageous to refer to CI rather than actions explicitly in case we change this again (which seems likely given actions is pretty slow).

That being said, I acknowledge that the implementation specifics here are tied to actions. Perhaps we can differentiate that externally to this class, things generally only care about CI in general whereas this class fulfills its contract by performing actions-specific parsing/checking. WDYT?

Reply:
Basically, mostly everywhere else in this PR that refers to actions probably doesn't need to. A litmus check is: does this line of code/function/class actually care whether we're using Circle, Travis, Actions, something else? I think most lines of code outside of this file don't care, and those lines should refer to CI generally instead of specifically actions.

Reply:
Thanks @BenHenning for explaining!